js/wshop.refresh.js

```diff
@@ -0,0 +1,178 @@
+// Update existing products
+// Create pages for new products
+// Publish if auto_approve is set to 'true'
+
+var wshopRefresh = {
+
+    vars: wshopVars,
+    shopClient: null,
+    productsLeft: 0,
+    processedIDs: [],
+
+    init: function(){
+        wshopRefresh.initShopify();
+        wshopRefresh.initListener();
+    },
+
+    initListener: function(){
+
+        jQuery('#wpshopify-refresh-button').on('click', function(e){
+            e.preventDefault();
+
+            // Cancel if already working
+            if( jQuery('#wpshopify-refresh-button').hasClass('disabled') ) return;
```

Comment:
You shouldn't need a 'disabled' class, you should be able to do any styling etc. with the disabled attribute.